src/app/code/community/Shopgate/Cloudapi/etc/config.xml

```diff
@@ -2,7 +2,7 @@
 <config>
     <modules>
         <Shopgate_Cloudapi>
-            <version>3.0.0</version>
+            <version>3.0.1</version>
```

Comment:
@augsteyer sure this number need to be increased for the install script to run, but won't travis take care of it within the deployment process?

Reply:
Ah I remember travis cannot change and push while releasing, so this one has to be set manually anyway. Usually this needs to be done as a release task, but as for this commit it's necessary to increase it already to make the install script running, I think we are good here...

Reply:
yes, you are correct. We would have to manually adjust these numbers before release in any case. So it's fine to do, as long as there is only 1 version change per cycle. So if the next PR needs to update a script too, it will have to remove the 3.0.1 resource from the database and add the script code to the existing one 3.0.1 upgrade script.

Reply:
that said, most likely this version will need to be 3.1.0 though. We are not releasing patch/fixes as we are working with a story that handles the 3.1.0 changes. Although, to be honest, versioning was not discussed much and probably should be addressed. We have 3.1.0 in the sprint tickets at the moment, so would it make sense to bundle all the upgrade scripts under the next big version release?

Reply:
It would make sense and it can be done afterwards adding a additional install script 3.1
Should be added as AC for the last story planned for 3.1, so please keep that in mind for next planning.